plugins/org.eclipse.n4js/src/org/eclipse/n4js/scoping/ExportedElementsCollector.java

```diff
@@ -169,7 +171,7 @@ private void doCollectElement(String exportedName, TExportableElement exportedEl
 						visibility.accessModifierSuggestion));
 			}
 		} else {
-			if (exportedElem instanceof Type) {
+			if (exportedElem instanceof Type && !(exportedElem instanceof TFunction)) {
```

Comment:
Is this change necessary? (now that `TFunction` no longer inherits from `Type`)

Reply:
As discussed, I will try later to change inheritance regarding `GenericType` inheriting from `Type`.